### DISCUSSION TITLE
Poor performance

### DISCUSSION THREAD

**Discussion by lpereira98** [MahApps/MahApps.Metro#4179] [Q&A]
Not sure if this is a problem of my PC or of the library, but it it's from my PC i would like to get some suggestions on how to solve it, all my graphic drives are updated.

Poor performance shown here:

https://user-images.githubusercontent.com/658431/131483631-94fd0124-fab4-46b9-afbe-defff0b2e4b1.mp4

## Steps to reproduce

My App.xaml:

```
<Application x:Class="TestEditor.App"
             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
             xmlns:local="clr-namespace:TestEditor"
             StartupUri="MainWindow.xaml">
    <Application.Resources>
        <ResourceDictionary>
            <ResourceDictionary.MergedDictionaries>
                <!-- MahApps.Metro resource dictionaries. Make sure that all file names are Case Sensitive! -->
                <ResourceDictionary Source="pack://application:,,,/MahApps.Metro;component/Styles/Controls.xaml" />
                <ResourceDictionary Source="pack://application:,,,/MahApps.Metro;component/Styles/Fonts.xaml" />
                 <!--Theme setting--> 
                <ResourceDictionary Source="pack://application:,,,/MahApps.Metro;component/Styles/Themes/Light.Blue.xaml" />
            </ResourceDictionary.MergedDictionaries>
        </ResourceDictionary>
    </Application.Resources>
</Application>
```

MainWindow.xaml:

```
<Controls:MetroWindow x:Class="TestEditor.MainWindow"
        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
        xmlns:Controls="http://metro.mahapps.com/winfx/xaml/controls"
        xmlns:local="clr-namespace:TestEditor"
        mc:Ignorable="d"
        Title="MainWindow" Height="450" Width="800">
    <Grid Name="gridMain_" Tag="abc">
        <Grid.RowDefinitions>
            <RowDefinition Height="Auto"/>
        </Grid.RowDefinitions>
        <Grid.ColumnDefinitions>
            <ColumnDefinition Width="*"/>
            <ColumnDefinition Width="*"/>
        </Grid.ColumnDefinitions>
        <Button Grid.Column="0" Grid.Row="0" Click="Button_Click" HorizontalAlignment="Center" Content="Click Me!" />
        <StackPanel Name="stackPanel_" Grid.Row="0" Grid.Column="1" />
    </Grid>
    
</Controls:MetroWindow>

```

## Environment

```
MahApps.Metro version: v2.4.7
Windows build number: Win10
Visual Studio: 2019 16.10.2
Target Framework: .NET 5.0
```

## Screenshots

![image](https://user-images.githubusercontent.com/44083899/131480603-ce2aac20-e662-47dc-992b-7165ecb7f98d.png)

**Comment by timunie** [comment]
Hi @lpereira98 

I have never seen this before and your Apps seems quite simple. Can you please perform the following?
1. Test it on any other PC 
2. Test it without MahApps but every thing else the same. Do have also trouble? 
3. If 1 and 2 are done and you still have trouble, please share your entire sample via GitHub. Maybe there is something in the code behind which triggers this behavior. 

Happy coding
Tim

**Comment by lpereira98** [comment]
Hi @timunie 

It's definitely some problem about my PC, this app runs like a charm in any other PC. And even without Metro, i still have problems, i just haven't noticed before.
I can't figure it out, my PC definitely satisfies WPF prerequisites.
Any idea of what can be the problem? My graphic drives are updated, windows is updated 🤷‍♂️

**Comment by timunie** [comment]
@lpereira98 I have no idea and this is also not the right place to discuss this. Please close the issue as it is nothing we can solve. 🙏 Thank you
